galaxy_api/api/ui/serializers/collection.py

```diff
@@ -82,7 +82,8 @@ class _CollectionSerializer(serializers.Serializer):
     name = serializers.CharField()
     download_count = serializers.IntegerField(default=0)
     latest_version = CollectionVersionSerializer(source='*')
-    is_certified = serializers.BooleanField()
+    certification = serializers.ChoiceField(
```

Comment:
Minor: Normally I would prefer enum for list of acceptable string values.

Otherwise LGTM.

Reply:
The documentation to [ChoiceField](https://www.django-rest-framework.org/api-guide/fields/#choicefield) referres to the parameter as a _list_, but surely I can try if _enum_ works too.

Reply:
@cutwater I've applied the suggestions to use enum and updated the PR. Please try again the changes if they work for you.

Reply:
@cutwater Taking back using enum